include/pagmo/population.hpp

```diff
@@ -79,7 +79,7 @@ class population
      * Constructs an empty population with a pagmo::null_problem.
```

Comment:
This is just a draft to clarify the structure and help in making the decision on wether we want it. If we agree I would produce a fully "functional" PR